README.md

```diff
@@ -52,26 +52,33 @@ where app.json may look something like:
             "sampleInterval": 1000
         }
     },
-    "plugins": {
-        "good": {
-            "opsInterval": 5000,
-            "requestHeaders": true,
-            "reporters": [{
-                "reporter": "good-console",
-                "events": { "response": "*", "ops": "*", "log": "*", "error": "*" }
-            },
-            {
-                "reporter": "good-file",
-                "events": { "response": "*", "error": "*" },
-                "config": "/log/response.log"
-            },
-            {
-                "reporter": "good-file",
-                "events": { "ops": "*" },
-                "config": "/log/ops.log"
-            }]
+    "registrations": [
```

Comment:
@geek: Please note that registration options (`once`, `select`, `routes`) need to be passed as well and they should not be in `options` as `options` are passed to the plugin.